ui/app/serializers/node.js

```diff
@@ -9,11 +11,10 @@ export default ApplicationSerializer.extend({
   },
 
   normalize(modelClass, hash) {
-    // Proxy local agent to the same proxy express server Ember is using
-    // to avoid CORS
-    if (this.get('config.isDev') && hash.HTTPAddr === '127.0.0.1:4646') {
-      hash.HTTPAddr = '127.0.0.1:4200';
-    }
+    // Transform the map-based Drivers object into an array-based ClientDriver fragment list
```

Comment:
This comment refers to `ClientDriver`, but I think the fragment model name is `NodeDriver`. Am I mistaken?

Reply:
Ugh, yeah. We use Node and Client sorta but not really interchangeably. I try to keep all the data layers stuff using Node and all the routes and copy saying Client. Really easy to get wires crossed.